packages/wonder-blocks-modal/components/modal-launcher.js

```diff
@@ -125,7 +125,11 @@ class ModalLauncherKeypressListener extends React.Component<{
     }
 
     _handleKeyup = (e: KeyboardEvent) => {
-        if (e.key === "Escape") {
+        const escape = 27;
+        const keyCode = e.which || e.keyCode;
```

Comment:
I thought `which` was only necessary for old versions of IE.

Reply:
I don't think so. From the errors I was seeting, `keyCode` is becoming obsolete in favour of `which`. In the end, I just copied what the modal did.